readme.txt

```diff
@@ -11,21 +8,30 @@ What is this game?
 Caffeine Fueled is a side-scroller game where you take control of a guy with a coffee-powered jetpack and try to dodge the evil copter kids and other random stuff flying in the sky. The game also features a shop where you can buy cosmetic hats and pets. 
 
 It's my entry for the Pygame Community New Years Jam, which can be found here:
-https://itch.io/jam/pygame-community-jam
+https://itch.io/jam/pygame-community-jam/
 
 Controls
 --------
-W, SPACE, UP 		- Move up
+W, SPACE, UP 		  - Move up
 S, SHIFT, DOWN 		- Move down
-A, LEFT 			- Move Left
-D, RIGHT 			- Move Right
+A, LEFT 			    - Move Left
+D, RIGHT 			    - Move Right
+
+Requeriments
```

Comment:
Oops, a typo here! It should be 'Requirements' !

Reply:
I will fix it, thanks.